README.md

```diff
@@ -1,56 +1,104 @@
+
+[Changelog]: docs/Changelog.md
+[Schematic]: docs/Schematic.pdf
+[Preview]: docs/Preview.png
+[Vendors]: docs/Vendors.md
+[PCB]: cad
+
+[Green Pro Micro]: http://www.40percent.club/2017/09/green-pro-micro.html
+[License]: https://creativecommons.org/licenses/by-sa/4.0/
+
+[Alvaro]: https://github.com/Ariamelon/Alvaro
+[Comet]: https://github.com/vattern/comet
+[Elite-C]: https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4
+[Goldfish-clone-JLCPCB]: https://github.com/JeremyJStarcher/Goldfish-clone-JLCPCB
```

Comment:
Can you move these links to the bottom of the file? The Markdown source reads much better that way